qt/aqt/main.py

```diff
@@ -1055,9 +1054,9 @@ def onDeckConf(self, deck=None):
         if not deck:
             deck = self.col.decks.current()
         if deck["dyn"]:
-            import aqt.dyndeckconf
+            import aqt
```

Comment:
This makes the pylint check fail. But without reimporting, I get an UnboundLocalError at runtime. Do you know why that is and how to solve it?

Reply:
Moving the import statement in the else clause to before the if should fix it - not exactly clear!